.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
what does this file do?

Reply:
I believe it holds configuration for prettier. But we might not even need to set any options. The defaults may be ok. 

Not even sure we'll want to use prettier but I figured it's worth a look.

https://prettier.io/